benchbuild/utils/actions.py

```diff
@@ -508,33 +514,31 @@ def __call__(self):
         self.status = StepResult.OK
         return results
 
-    def __str__(self, indent=0):
-        sub_actns = [a.__str__(indent + 1) for a in self.actions]
-        sub_actns = "\n".join(sub_actns)
+    def __str__(self, indent: int = 0) -> str:
+        sub_actns = "\n".join([a.__str__(indent + 1) for a in self.actions])
```

Comment:
E1133 (not-an-iterable): Non-iterable value self.actions is used in an iterating context  <!-- Automatically posted by Lintly -->